libs/pages/login/feature/src/lib/login.tsx

```diff
@@ -3,6 +3,7 @@ import { useNavigate } from 'react-router'
 import { LayoutLogin, Login } from '@console/pages/login/ui'
 import { ONBOARDING_URL, useAuth, useDocumentTitle, AuthEnum, OVERVIEW_URL } from '@console/shared/utils'
 import { useOrganization } from '@console/domains/organization'
+import posthog from "posthog-js";
```

Comment:
Same here

Reply:
Don't we have the pre-commit script that apply prettier to avoid this kind of things?